validate/main.go

```diff
@@ -15,29 +15,32 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"strings"
 
 	config "github.com/coreos/ignition/config/v3_0_experimental"
 	"github.com/coreos/ignition/internal/version"
-
-	"github.com/spf13/cobra"
 )
 
 var (
 	flagVersion bool
-	rootCmd     = &cobra.Command{
-		Use:   "ignition-validate config.ign",
-		Short: "ignition-validate will validate Ignition configs",
-		Run:   runIgnValidate,
-	}
 )
 
+func init() {
+	flag.BoolVar(&flagVersion, "version", false, "print the version of ignition-validate")
+	flag.Usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage:\n  %s config.ign [flags]\n\n", os.Args[0])
```

Comment:
go 1.8 doesn't have `flag.CommandLine.Output()`, since we're not setting it, I'd just hardcode stderr

Reply:
I actually did `stderr`, and then changed my mind to use `flag.CommandLine.Output()`

Reply:
We initially had 1.8.x support because we weren't sure what RHCOS would require. Looks like 1.10 is fine so I've opened https://github.com/coreos/ignition/pull/704. Once it merges, you can rebase onto it and it shold clear up the CI failure and we can merge.